sapi/fpm/fpm/fpm_status.c

```diff
@@ -385,6 +385,77 @@ int fpm_status_handle_request(void) /* {{{ */
 				full_post = "]}";
 			}
 
+			/* Prometheus */
+		} else if (fpm_php_get_string_from_table(_GET_str, "prometheus")) {
+			sapi_add_header_ex(ZEND_STRL("Content-Type: text/plain"), 1, 1);
+			time_format = "%s";
+
+			short_syntax =
+				"# HELP phpfpm_up Could PHP-FPM be reached?\n"
+                "# TYPE phpfpm_up gauge\n"
+                "phpfpm_up{pool=%1$s} 1\n"
+                "# HELP phpfpm_start_time When FPM has started.\n"
+                "# TYPE phpfpm_start_time gauge\n"
+				"phpfpm_start_since{pool=%1$s} %3$lu\n"
+                "# HELP phpfpm_start_since The number of seconds since FPM has started.\n"
+                "# TYPE phpfpm_start_since counter\n"
+				"phpfpm_start_since{pool=%1$s} %4$lu\n"
+                "# HELP phpfpm_accepted_connections The number of requests accepted by the pool.\n"
+                "# TYPE phpfpm_accepted_connections counter\n"
+				"phpfpm_accepted_connections{pool=%1$s} %5$lu\n"
+                "# HELP phpfpm_listen_queue The number of requests in the queue of pending connections.\n"
+                "# TYPE phpfpm_listen_queue gauge\n"
+				"phpfpm_listen_queue{pool=%1$s} %6$lu\n"
+                "# HELP phpfpm_max_listen_queue The maximum number of requests in the queue of pending connections since FPM has started.\n"
+                "# TYPE phpfpm_max_listen_queue counter\n"
+				"phpfpm_max_listen_queue{pool=%1$s} %7$d\n"
+                "# TYPE phpfpm_listen_queue_length gauge\n"
+				"# HELP phpfpm_listen_queue_length The size of the socket queue of pending connections.\n"
```

Comment:
indent

Reply:
Yes indents look weird, will be sure to fix them. They are fine locally some how 0_O